packages/oui-navbar/_mixins.less

```diff
@@ -369,6 +375,47 @@
         outline-color: initial;
         outline-offset: -2px;
       }
+
+      .notification__icon {
+        margin: rem-calc(3) rem-calc(10) 0 0;
+      }
+
+      .notification__icon > i {
```

Comment:
Here I would have called the class "oui-icon" because like that, you forced people to call a <i> but some people prefer using a span. And it could potentially apply to others <i> that isn't the one you wanted to.